.ci/ansible/tasks/install_ssh_keys.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: Install SSH keys
+  shell: |
+    /home/{{ansible_user}}/e2e-testing/.ci/scripts/import-ssh-keys.sh
```

Comment:
Use `become: false` here I think the authorized keys file is being owned by root